src/metabase/db/custom_migrations.clj

```diff
@@ -1202,7 +1202,7 @@
                (no-db?))
       (let [table-name->raw-rows  (load-edn "sample-content.edn")
             example-dashboard-id  1
-            example-collection-id 1
+            example-collection-id 2 ; trash collection is 1
```

Comment:
The trash collection was introduced here 
https://github.com/metabase/metabase/blob/a4da2963b305fcbc79ac68063dbf36662a6c3f39/resources/migrations/001_update_migrations.yaml#L7242-L7254